src/org/loklak/api/aggregation/ClassifierServlet.java

```diff
@@ -80,6 +80,7 @@ public JSONObject serviceImpl(Query call, HttpServletResponse response, Authoriz
         // Classifier and classes
         String classifier = call.get("classifier", "").toLowerCase();
```

Comment:
getCallValues() can be set up and used, and all this data can be fetched and worked on in other method in next PRs.